### ISSUE TITLE
Prebuilt jekyll zip hosted on GitHub codeload is missing mathjax include file

### ISSUE THREAD

**Issue by sgurungp** [daattali/beautiful-jekyll#1360]
Users that use GitHub Pages and beautiful-jekyll as a remote theme (`remote_theme: daattali/beautiful-jekyll@6.0.1` in `_config.yml`) may want to take advantage of MathJax support, e.g. in their posts. I believe that there may be a bug caused by a missing file here.

In classic beautiful-jekyll, you would make sure to have `_includes/mathjax.html` in your fork of the repo, just like all the other includes. I also note that `_includes/head.html` references mathjax.html from (the commit for) #195 .

In sites built by GitHub Actions for GitHub Pages, under the covers, it is downloading https://codeload.github.com/daattali/beautiful-jekyll/zip/6.0.1 (strictly: beautiful-jekyll-6.0.1.zip), unzipping it in a container and then building the site. All of this works fine...except that this zip file does not include `mathjax.html` in its _includes folder, nor the same version of `head.html` that would refer to it.

As a result, mathjax support is not available to the user (at least, it seems to me).

I speculate that that the fix is to regenerate https://codeload.github.com/daattali/beautiful-jekyll/zip/6.0.1 from the same commit point as originally addresses #195 so that the zipfile includes the right head.html and mathjax.html.

**Comment by daattali** [comment]
Official mathjax support was only added after v6.0.1

You can see the commit here https://github.com/daattali/beautiful-jekyll/commit/49b2d8b7a0a110b40f11679f5a26bc299660ec90

If you use `remote_theme: daattali/beautiful-jekyll` (without specifying a version) it should use the latest unreleased version which should have mathjax. In version 6.0.1 I didn't have mathjax support yet so you would need to add it yourself manually using the tips that are provided in #195
